pages/partials/mathjax.ejs

```diff
@@ -1,50 +1,50 @@
 <script>
 
- // default to CHTML because of https://github.com/PrairieLearn/PrairieLearn/pull/3520
- var outputComponent = 'output/chtml';
- <% if (userAgent && userAgent.engine.name == 'WebKit') { %>
-     // workaround for https://github.com/PrairieLearn/PrairieLearn/issues/3540
-     outputComponent = 'output/svg';
- <% } %>
- <% if (typeof questionHtml != 'undefined' && questionHtml.includes('pl-requires-svg-mathjax')) { %>
-     outputComponent = 'output/svg';
- <% } %>
- <% if (userAgent && userAgent.os.name == 'Linux') { %>
-     // Workaround for horizontal lines disappearing
-     // on Linux. Seems to work in Firefox, Brave,
-     // Chromium.
-     outputComponent = 'output/svg';
- <% } %>
+// default to CHTML because of https://github.com/PrairieLearn/PrairieLearn/pull/3520
+var outputComponent = 'output/chtml';
+<% if (userAgent && userAgent.engine.name == 'WebKit') { %>
+    // workaround for https://github.com/PrairieLearn/PrairieLearn/issues/3540
+    outputComponent = 'output/svg';
+<% } %>
+<% if (typeof questionHtml != 'undefined' && questionHtml.includes('pl-requires-svg-mathjax')) { %>
+    outputComponent = 'output/svg';
+<% } %>
+<% if (userAgent && userAgent.os.name == 'Linux') { %>
+    // Workaround for horizontal lines disappearing
+    // on Linux. Seems to work in Firefox, Brave,
+    // Chromium.
+    outputComponent = 'output/svg';
+<% } %>
 
- var MathJax = {
-     tex: {
-         inlineMath: [['$', '$'], ['\\(', '\\)']]
-     },
-     svg: {
-         fontCache: 'global'
-     },
-     loader: {
-         load: ['input/tex', outputComponent]
-     },
-     // Register a callback to be run when MathJax is loaded, use MathJax.config.onReady().
-     readyQueue: [],
-     onReady: (cb) => { MathJax.config.readyQueue.push(cb); },
-     startup: {
-         ready: () => {
-             MathJax.startup.defaultReady();
-             MathJax.Hub = {
-                 Queue: function(){
-                     MathJax.typeset();
-                     console.warn("MathJax.Hub.Queue() has been deprecated in 3.0, please use MathJax.typeset() or MathJax.typesetPromise()");
-                 }
-             };
-         },
-         pageReady: () => {
-             MathJax.startup.defaultPageReady();
-             MathJax.config.onReady = cb => cb();
-             MathJax.config.readyQueue.forEach(cb => cb());
-         }
-     }
- };
+var MathJax = {
+    tex: {
+        inlineMath: [['$', '$'], ['\\(', '\\)']]
+    },
+    svg: {
+        fontCache: 'global'
+    },
+    loader: {
+        load: ['input/tex', 'ui/menu', outputComponent]
```

Comment:
This is the only meaningful change here; the rest of this diff is just fixing indentation.

Reply:
Yep, I used the very helpful "no whitespace" option in GH when reviewing this.